### PR TITLE
Link build results to testgrid (fixes #272).

### LINKED ISSUE #272: Gubernator should link to jobs to testgrid
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-e2e-gce/20010/ shows a link up at the top to kubernetes-e2e-gce... unfortunately it links to the storage browser rather than http://testgrid.k8s.io/google-gce

We should make it link to testgrid instead.

### PR DESCRIPTION
**Pull request by rmmh**
This makes it easy to view historical failures for a job or test.

The neatest / grossest thing here is in `pb_glance.py` -- a tiny protobuf parser in 40 lines of code. See `pr_glance_test.py` for more usage information.

### CODE REVIEW

File: gubernator/view_build.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ class BuildHandler(view_base.BaseHandler):
     def get(self, prefix, job, build):
         self.check_bucket(prefix)
         job_dir = '/%s/%s/' % (prefix, job)
+        testgrid_query = testgrid.path_to_query(job_dir[1:-1])
```
Comment (spxtr) [inline gubernator/view_build.py]: Why `[1:-1]`?

Comment (rmmh) [inline gubernator/view_build.py]: testgrid wants paths like 'kubernetes-jenkins/jobs/gce-e2e'

File: gubernator/testgrid.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        },
+        2: 'name',
+    }
+}
+
+_testgrid_config = None
+
+
+def get_config():
+    global _testgrid_config  # pylint: disable=global-statement
```
Comment (spxtr) [inline gubernator/testgrid.py]: Not a huge fan of this global.

File: gubernator/testgrid.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            12: {},
+        },
+        2: 'name',
+    }
+}
+
+_testgrid_config = None
+
+
+def get_config():
```
Comment (spxtr) [inline gubernator/testgrid.py]: Comments would be appreciated for the three functions in this module, even though they're mostly self-explanatory.

File: gubernator/testgrid.py (hunk 2 above)
Comment (rmmh) [inline gubernator/testgrid.py]: Do you have any preferences for alternatives?

File: gubernator/testgrid.py (hunk 3 above)
Comment (rmmh) [inline gubernator/testgrid.py]: Added docstrings for these functions.

File: gubernator/testgrid.py (hunk 2 above)
Comment (spxtr) [inline gubernator/testgrid.py]: If you need it to be loaded once and then used multiple times then load it once at the start and then pass it to whoever uses it.

**Pull request merged**